invenio_rdm_records/dumpers.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: I would structure the module like Invenio-Records-Resources:

```
invenio_rdm_records/records/api
invenio_rdm_records/records/dumpers.py
invenio_rdm_records/records/models.py
invenio_rdm_records/resources/...
invenio_rdm_records/services/...
```

This way it's clear what belongs to which layer.

Reply:
Shelved: We should think about where to put these tools. The EDTF dumper is likely to be useful outside of RDM.

Reply:
Refactor done.

Extension could potentially be moved to invenio-records